apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -20,6 +20,8 @@ export function Viewer() {
   const { client: authClient } = useAuthorizationContext();
   const viewCreatorOptions = useViewCreatorOptions();
   const { iTwinId, iModelId } = useIModelInfo();
+  const [BingMapsKey] = useState(process.env.IMJS_BING_MAPS_KEY ?? "");
+  const [CesiumKey] = useState(process.env.IMJS_CESIUM_ION_KEY ?? "");
```

Comment:
Looks like these can be moved outside `Viewer` component and don't need to be stored in React state.